PowerShell/BuildingBlocks/ServicesAndTasks/DeployRunbook.ps1

```diff
@@ -0,0 +1,16 @@
+﻿# Initialize variables
+$offeringMetadataId = [GUID](Read-Host -Prompt 'Runbook ID')
+$customerId = [GUID](Read-Host -Prompt 'Customer ID')    
+
+# Authenticate
```

Comment:
In other scripts, credentials are requested first, then customer id. Could be useful to normalize here as well.

Reply:
Good catch, fixed